src/declarations/stencil-public-docs.ts

```diff
@@ -1,3 +1,15 @@
+import {
+  ComponentCompilerEventComplexType,
+  ComponentCompilerMethodComplexType,
+  ComponentCompilerPropertyComplexType,
+  ComponentCompilerReferencedType,
+} from './stencil-private';
+
+/**
+ * TODO: Document
+ */
+export type JsonDocsTypeLibrary = Record<string, ComponentCompilerReferencedType>;
```

Comment:
Open to other names for the type here, I sort of went with what I thought was the most obvious candidate